routes/auth.js

```diff
@@ -0,0 +1,146 @@
+const { mail } = require("./mail/mail");
+const db = require("../db/models");
+const { generateTokens } = require("./auth/token");
+const hostToDoList = process.env.FELLDEK_HOST_TO_DO_LIST;
+
+let auth = {};
+let a = { body: { email: "123test", password: "123" } };
+
+auth.signUp = async (req, res) => {
+  try {
+    const user = await db.users.create({
+      email: req.body.email,
+      password: req.body.password,
+    });
+    await auth.signIn(req, res);
+    await mail.confirmEmail({ id: user.dataValues.id, email: req.body.email });
+  } catch (e) {
+    console.log("func signUp", e);
+    if (e.original.code === "23505") {
+      res.status(500).json({
+        error: true,
+        authorization: false,
+        description: "This E-mail already registered",
+      });
+      console.log("This E-mail already registered");
+    } else {
+      res.status(500).json({
+        error: true,
+        authorization: false,
+        description: e.errors[0].message,
+      });
+    }
+  }
+};
+
+auth.confirmEmail = async (req, res) => {
+  try {
+    await db.users.update({ confirm: true }, { where: { id: req.params.id } });
+    res.redirect(hostToDoList);
```

Comment:
Logic for redirect should be handled on FE. Send status code/custom response and handle it on FE

Reply:
Please read more about REST API

Reply:
Implement user verification using token